openlibrary/components/MergeUI/MergeTable.vue

```diff
@@ -51,7 +51,7 @@
     <tfoot>
       <MergeRow
         v-if="merge"
-        :record="merge.record"
+        :record="enhancedMergeRecord"
         :selected="selected"
         :fields="fields"
         :merged="merge"
```

Comment:
I believe this needs to forward the full `enhancedMerge` result to avoid causing some downstream checks to fail.

```suggestion
        :record="enhancedMerge.record"
        :selected="selected"
        :fields="fields"
        :merged="enhancedMerge"
```

We're seeing some unexpected loading indicators popping up:

![image](https://github.com/user-attachments/assets/9362ac9c-8a5a-4fac-8b0e-6d0fa7cea2b9)

Note you'll have to update the `enhancedMergeRecord` computed to return the full merge object, not just the record.